src/HTML.js

```diff
@@ -439,7 +445,8 @@ export default class HTML extends PureComponent {
                         parentIndex,
                         key,
                         data,
-                        rawChildren: children
+                        rawChildren: children,
+                        ...renderersProps
```

Comment:
I don't see how that fixes the issue because `textSelectable` is already merged into this last renderer argument [from the `...props` spread, L441](https://github.com/archriss/react-native-render-html/blob/d6177a64d838399dff2a76575d9c0f201071bb1f/src/HTML.js#L441), [defined L399](https://github.com/archriss/react-native-render-html/blob/d6177a64d838399dff2a76575d9c0f201071bb1f/src/HTML.js#L399). Did I miss something? Can you explain your rationale?